home05/js/script_1.js

```diff
@@ -0,0 +1,43 @@
+// Создать объект, описывающий автомобиль (производитель, модель, год выпуска, средняя скорость)
+
+const auto = {
+    manufacturer: "Toyota",
+    modelAuto: "Corolla",
+    yearCreate: 2010,
+    averageSpeed: 120
+}
+// Функция для вывода на экран информации об автомобиле;
+function viewerObjAuto(n1) {
+    let result = '';
+    for (const key in auto) {
+        console.log(`${key}: ${auto[key]}`);
+        result += `${key}: ${auto[key]} \n`;
+    }
+    return result;
+}
```

Comment:
Аргумент у тебя `n1`, а показываешь ты обьект `auto` - извне. 
```suggestion
function viewerObjAuto(auto) {
    let result = '';
    for (const key in auto) {
        result += `${key}: ${auto[key]} \n`;
    }
    return result;
}
```